tensorflow/core/common_runtime/mkl_layout_pass_test.cc

```diff
@@ -188,27 +189,6 @@ REGISTER_OP("BFloat16Output2")
     .SetIsStateful();
 #endif  // ENABLE_INTEL_MKL_BFLOAT16
 
-/////////////////////////////////////////////////////////////////////
-// Macros for handling registeration for various types
-/////////////////////////////////////////////////////////////////////
-
-#define REGISTER_TEST_FLOAT32(TEST) REGISTER_TEST(TEST, DT_FLOAT, Float32Input);
-
-#ifdef ENABLE_INTEL_MKL_BFLOAT16
-#define REGISTER_TEST_BFLOAT16(TEST) \
-  REGISTER_TEST(TEST, DT_BFLOAT16, BFloat16Input);
-
-#define REGISTER_TEST_ALL_TYPES(TEST) \
-  REGISTER_TEST_FLOAT32(TEST);        \
-  REGISTER_TEST_BFLOAT16(TEST);
-#else
-#define REGISTER_TEST_ALL_TYPES(TEST) REGISTER_TEST_FLOAT32(TEST);
-#endif  // ENABLE_INTEL_MKL_BFLOAT16
-
-/////////////////////////////////////////////////////////////////////
-//  Unit tests related to node merge optimization
-/////////////////////////////////////////////////////////////////////
```

Comment:
I think these are supposed to remain in this file. Would you mind moving it back?

Reply:
Thanks for the reminder! I have moved it back.